app/lib/shared/markdown.dart

```diff
@@ -6,28 +6,22 @@ import 'package:logging/logging.dart';
 import 'package:markdown/markdown.dart' as m;
 import 'package:pana/pana.dart' show getRepositoryUrl;
 import 'package:path/path.dart' as p;
+import 'package:sanitize_html/sanitize_html.dart';
 
 final Logger _logger = Logger('pub.markdown');
 
-List<m.InlineSyntax> _inlineSyntaxes;
-
-void _initInlineSyntaxes() {
-  if (_inlineSyntaxes != null) return;
-  _inlineSyntaxes = m.ExtensionSet.gitHubWeb.inlineSyntaxes
-      .where((s) => s is! m.InlineHtmlSyntax)
-      .toList();
-  _inlineSyntaxes.add(_InlineBrHtmlSyntax());
-}
+const _whitelistedClassNames = <String>[
+  'hash-header',
+  'hash-link',
```

Comment:
How about `language-dart` and `language-yaml`.... maybe even `language-<whatever>` to allow for syntax highlighting?

Reply:
Good catch! Coincidentally (as we were talking about tests), this was caught by the templates_test.dart too :)

Reply:
FYI – could be a Set instead of a list 😄

Reply:
ah, sorry, missed this comment... not sure though if it will be faster with only two items in it...

Reply:
🤷‍♂